docs/UsersGuide/source/BackgroundInfo/Introduction.rst

```diff
@@ -91,44 +103,45 @@ A list of available component documentation is shown in :numref:`Table %s <list_
 
 .. _list_of_documentation:
 
-.. list-table::  Centralized list of documentation
+.. list-table:: Centralized List of Documentation
    :widths: 20 50
    :header-rows: 1
 
    * - Documentation
      - Location
-   * - HPC-Stack Documentation
-     - https://hpc-stack.readthedocs.io/en/latest/
    * - spack-stack Documentation
-     - https://spack-stack.readthedocs.io/en/latest/
+     - https://spack-stack.readthedocs.io/en/1.4.1/
+   * - HPC-Stack Documentation
+     - https://hpc-stack-epic.readthedocs.io/en/release-srw-public-v2.2.0/
    * - UFS_UTILS Technical Documentation
-     - https://noaa-emcufs-utils.readthedocs.io/en/latest
+     - https://noaa-emcufs-utils.readthedocs.io/en/ufs_utils_1_11_0/
    * - UFS_UTILS Scientific Documentation
-     - https://ufs-community.github.io/UFS_UTILS/index.html
+     - https://ufs-community.github.io/UFS_UTILS/ver-1.11.0/index.html
    * - UFS Weather Model User's Guide
-     - https://ufs-weather-model.readthedocs.io/en/latest
+     - https://ufs-weather-model.readthedocs.io/en/ufs-srw-v2.2.0-doc/
    * - FV3 Technical Documentation
      - https://noaa-emc.github.io/FV3_Dycore_ufs-v2.0.0/html/index.html
    * - FV3 Scientific Documentation
      - https://repository.library.noaa.gov/view/noaa/30725
    * - CCPP Technical Documentation
-     - https://ccpp-techdoc.readthedocs.io/en/latest/
+     - https://ccpp-techdoc.readthedocs.io/en/ufs_srw_app_v2.2.0/
    * - CCPP Scientific Documentation
      - https://dtcenter.ucar.edu/GMTB/UFS_SRW_App_v2.2.0/sci_doc/index.html
    * - Stochastic Physics Documentation
-     - https://stochastic-physics.readthedocs.io/en/latest/
+     - https://stochastic-physics.readthedocs.io/en/release-public-v3/
    * - ESMF manual
-     - https://earthsystemmodeling.org/docs/release/latest/ESMF_usrdoc/
+     - https://earthsystemmodeling.org/docs/release/ESMF_8_4_2/ESMF_usrdoc/
    * - Unified Post Processor User's Guide
-     - https://upp.readthedocs.io/en/latest/
+     - https://upp.readthedocs.io/en/upp-srw-v2.2.0-docs/
    * - Unified Post Processor Scientific Documentation
      - https://noaa-emc.github.io/UPP/
    * - Unified Workflow User's Guide
-     - https://unified-workflow.readthedocs.io/en/latest/
+     - https://unified-workflow.readthedocs.io/en/v1.0.0/
    * - METplus User's Guide
-     - https://metplus.readthedocs.io/en/latest/Users_Guide/index.html
-   * - HEMCO User's Guide (a component of the NEXUS AQM system)
-     - https://hemco.readthedocs.io/en/stable/
+     - https://metplus.readthedocs.io/en/main_v4.1/Users_Guide/index.html
+
+.. COMMENT: Check links for stochastic_physics, CCPP Tech docs, UFS_UTILS
+
 
```

Comment:
@gspetro-NOAA - this is a great summary!

Reply:
Thanks! :)